helper/logging/null_logger.go

```diff
@@ -0,0 +1,43 @@
+package logging
```

Comment:
I actually suggest submitting a PR to go-hclog to move this upstream (async) as it's often useful to expose this from a logging package.

Reply:
I agree! I'll do that.